config/advisory_templates.yml

```diff
@@ -295,4 +297,44 @@ boilerplates:
       solution: |
         For MicroShift 4.{MINOR}.{PATCH}, read the following documentation, which will be updated shortly for this release, for important instructions on how to install the latest RPMs and fully apply this asynchronous errata update:
         
-        https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.{MINOR}/html/release_notes/index 
+        https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.{MINOR}/html/release_notes/index
+  bootimage:
+    synopsis: "OpenShift Container Platform 4.18.z packages update"
```

Comment:
i see hardcoded x.y.z values but we'd want to template these

Reply:
apologies - realized after I pinged you